tests/test_chordable_django_backend.py

```diff
@@ -147,7 +147,7 @@ def test_failing_chord_propagate():
 
     assert chord_data.callback_result.status == FAILURE
     assert isinstance(chord_data.callback_result.result, ChordError)
-    assert len(chord_data.callback_result.traceback) > 0
+    assert True, chord_data.callback_result.traceback
```

Comment:
This changes the assertion to always pass, did you want `assert chord_data.callback_result.traceback` instead?